class MyException(Exception): pass
class RepoException(MyException): pass
